dbms/src/AggregateFunctions/AggregateFunctionTopK.h

```diff
@@ -119,7 +127,14 @@ class AggregateFunctionTopK
 /// Generic implementation, it uses serialized representation as object descriptor.
 struct AggregateFunctionTopKGenericData
 {
-    using Set = SpaceSaving<StringRef, StringRefHash>;
+    using Set = SpaceSaving
+    <
+        std::string,
+        StringRef,
+        StringRefHash,
+        HashTableGrower<TOP_K_DEFAULT_DEGREE>,
+        HashTableAllocatorWithStackMemory<sizeof(StringRef) * (1 << (TOP_K_DEFAULT_DEGREE/2))>
```

Comment:
Ah, you're right, I confused degree with cell count passed to `set()`. I'll fix it in the next PR, 16 seems like a good default value (degree = 4).